[2022-07-28 10:21:12,129] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2022-07-28T10:21:11.212618+00:00 [queued]>
[2022-07-28 10:21:12,138] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2022-07-28T10:21:11.212618+00:00 [queued]>
[2022-07-28 10:21:12,138] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 10:21:12,139] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-28 10:21:12,139] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 10:21:12,153] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_table> on 2022-07-28 10:21:11.212618+00:00
[2022-07-28 10:21:12,157] {standard_task_runner.py:52} INFO - Started process 1295 to run task
[2022-07-28 10:21:12,160] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2022-07-28T10:21:11.212618+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpm8bkjk00', '--error-file', '/tmp/tmp0o6rsnr5']
[2022-07-28 10:21:12,161] {standard_task_runner.py:80} INFO - Job 36: Subtask create_table
[2022-07-28 10:21:12,237] {task_command.py:369} INFO - Running <TaskInstance: user_processing.create_table manual__2022-07-28T10:21:11.212618+00:00 [running]> on host 890323d6f4a0
[2022-07-28 10:21:12,313] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T10:21:11.212618+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-28T10:21:11.212618+00:00
[2022-07-28 10:21:12,322] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-07-28 10:21:12,326] {dbapi.py:208} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS users (

            firstname TEXT NOT NULL,
            lastname TEXT NOT NULL,
            country TEXT NOT NULL,
            username TEXT NOT NULL,
            password TEXT NOT NULL,
            email TEXT NOT NULL
        );
           , parameters: None
[2022-07-28 10:21:12,327] {postgres.py:94} INFO - NOTICE:  relation "users" already exists, skipping

[2022-07-28 10:21:12,339] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20220728T102111, start_date=20220728T102112, end_date=20220728T102112
[2022-07-28 10:21:12,373] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 10:21:12,417] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
